contracts/utils/contracts/src/v08/AuthorizableV08.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+
+  Copyright 2020 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.8;
+
+import "./interfaces/IAuthorizableV08.sol";
+import "./errors/LibRichErrorsV08.sol";
+import "./errors/LibAuthorizableRichErrorsV08.sol";
+import "./OwnableV08.sol";
+
+contract AuthorizableV08 is OwnableV08, IAuthorizableV08 {
+    /// @dev Only authorized addresses can invoke functions with this modifier.
+    modifier onlyAuthorized() {
+        _assertSenderIsAuthorized();
+        _;
+    }
+
+    // @dev Whether an address is authorized to call privileged functions.
+    // @param 0 Address to query.
+    // @return 0 Whether the address is authorized.
+    mapping(address => bool) public override authorized;
+    // @dev Whether an address is authorized to call privileged functions.
+    // @param 0 Index of authorized address.
+    // @return 0 Authorized address.
+    address[] public override authorities;
+
+    /// @dev Initializes the `owner` address.
+    constructor() OwnableV08() {}
+
+    /// @dev Authorizes an address.
+    /// @param target Address to authorize.
+    function addAuthorizedAddress(address target) external override onlyOwner {
+        _addAuthorizedAddress(target);
+    }
+
+    /// @dev Removes authorizion of an address.
+    /// @param target Address to remove authorization from.
+    function removeAuthorizedAddress(address target) external override onlyOwner {
+        if (!authorized[target]) {
+            LibRichErrorsV08.rrevert(LibAuthorizableRichErrorsV08.TargetNotAuthorizedError(target));
+        }
+        for (uint256 i = 0; i < authorities.length; i++) {
+            if (authorities[i] == target) {
+                _removeAuthorizedAddressAtIndex(target, i);
+                break;
+            }
+        }
```

Comment:
Generally speaking, this is an antipattern. I don't see this as a must-fix because `removeAuthorizedAddressAtIndex` exists, but this function `removeAuthorizedAddress` should be deprecated